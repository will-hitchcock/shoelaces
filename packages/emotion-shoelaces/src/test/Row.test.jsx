import React from 'react';
import { shallow } from 'enzyme';
import { Row } from '..';

describe('<Row />', () => {
  it('should render a <Row />', () => {
    expect(shallow(<Row />)).toMatchSnapshot();
  });
});
