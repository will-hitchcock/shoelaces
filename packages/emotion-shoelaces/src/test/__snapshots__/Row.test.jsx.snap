// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> should render a <Row /> 1`] = `
<styled.div
  _css={
    Object {
      "alignContent": "flex-start",
      "alignItems": "stretch",
      "alignSelf": "auto",
      "boxSizing": "border-box",
      "display": "flex",
      "flex": "0 1 auto",
      "flexWrap": "wrap",
      "justifyContent": "space-between",
      "marginLeft": "-0.5em",
      "marginRight": "-0.5em",
    }
  }
/>
`;
